From a5f9cfecf8a0548ed09886602cccf9a049b007ca Mon Sep 17 00:00:00 2001
From: Archana Shinde1 <archana.shinde2504@gmail.com>
Date: Wed, 20 Dec 2023 15:56:30 +0000
Subject: [PATCH] Updated protobuf to 4.21

---
 WORKSPACE | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/WORKSPACE b/WORKSPACE
index c60b5d8f..395f3ed3 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -91,6 +91,15 @@ switched_rules_by_language(
     grpc = True,
 )
 
+http_archive(
+    name = "com_google_protobuf",
+    sha256 = "f7042d540c969b00db92e8e1066a9b8099c8379c33f40f360eb9e1d98a36ca26",
+    strip_prefix = "protobuf-3.21.12",
+    urls = [
+        "https://github.com/protocolbuffers/protobuf/archive/v3.21.12.zip",
+    ]
+)
+
 http_archive(
     name = "org_tensorflow",
     sha256 = "ce357fd0728f0d1b0831d1653f475591662ec5bca736a94ff789e6b1944df19f",
-- 
2.40.1

