From 021ab7f7885825f05a7aedcc0c9f20812bcefdc6 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23in.ibm.com>
Date: Wed, 29 Mar 2023 08:41:52 +0000
Subject: [PATCH] Updated protobuf to 3.21.12

---
 WORKSPACE | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/WORKSPACE b/WORKSPACE
index ca83cb17..86081098 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -112,6 +112,15 @@ load("@lint_dependencies//:requirements.bzl", "pip_install")
 
 pip_install()
 
+http_archive(
+    name = "com_google_protobuf",
+    sha256 = "f7042d540c969b00db92e8e1066a9b8099c8379c33f40f360eb9e1d98a36ca26",
+    strip_prefix = "protobuf-3.21.12",
+    urls = [
+        "https://github.com/protocolbuffers/protobuf/archive/v3.21.12.zip",
+    ]
+)
+
 http_archive(
     name = "org_tensorflow",
     sha256 = "c030cb1905bff1d2446615992aad8d8d85cbe90c4fb625cee458c63bf466bc8e",
-- 
2.34.1

