From e2cb4b0eb8e728460ea0c4ba0e1d925edd2a77c3 Mon Sep 17 00:00:00 2001
From: Archana-Shinde1 <archana.shinde1@ibm.com>
Date: Fri, 1 Mar 2024 09:52:05 +0000
Subject: [PATCH] Update

---
 WORKSPACE | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index c60b5d8f..15d87c1d 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -102,11 +102,15 @@ http_archive(
 
 http_archive(
     name = "rules_python",
-    sha256 = "84aec9e21cc56fbc7f1335035a71c850d1b9b5cc6ff497306f84cced9a769841",
-    strip_prefix = "rules_python-0.23.1",
-    url = "https://github.com/bazelbuild/rules_python/releases/download/0.23.1/rules_python-0.23.1.tar.gz",
+    sha256 = "9acc0944c94adb23fba1c9988b48768b1bacc6583b52a2586895c5b7491e2e31",
+    strip_prefix = "rules_python-0.27.0",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.27.0/rules_python-0.27.0.tar.gz",
 )
 
+load("@rules_python//python:repositories.bzl", "py_repositories")
+
+py_repositories()
+
 load("@rules_python//python:repositories.bzl", "python_register_toolchains")
 load(
     "@org_tensorflow//tensorflow/tools/toolchains/python:python_repo.bzl",
-- 
2.40.1

